src/components/api/MaterialIconApi.js

```diff
@@ -7,6 +7,7 @@ import { materialFolderNamesToIcons } from "../gen/materialFolderIconsName";
 import Icons from "../icons/index";
 
 export const getMaterialFileIcon = (fileName) => {
+	fileName = fileName.toLowerCase();
```

Comment:
Add this for VS icons too.

Reply:
Added in https://github.com/minhazmiraz/file-extension-icon-JS/pull/4/commits/1c8b603de41b956be4b3b306b8e9e7363c58eb8a